buku.py

```diff
@@ -3259,6 +3294,22 @@ def main():
         # Enable color in logs
         setup_logger(logger)
 
+    # Set colors
+    if args.colorstr:
+        ID_str = '%d. %s [%s]\n'
+        ID_DB_str = '%d. %s'
+        MUTE_str = '%s (L)\n'
+        URL_str = '%s   > %s\n'
+        DESC_str = '%s   + %s\n'
+        TAG_str = '%s   # %s\n'
+        colors = Colors(*[COLORMAP[c] for c in args.colorstr], reset=COLORMAP['x'])
+        ID_DB_str = colors.ID_DB_str + ID_DB_str + COLORMAP['x']
```

Comment:
Can we have something similar to googler?
ID - bold  cyan (`G`)
Title - bold bright green (`K`)
DB Index (within []) - `\x1b[2m` (keep it hardcoded to dim)
URL - bright yellow (`l`)
comments/notes/desc - normal (`x`)
tags - blue (`e`)

Testing: Add a bookmark with url, title, tags and comment. Then
1. print it
2. search for it

to ensure your changes are good.